.github/actions/setup-etcd/action.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Setup etcd'
+description: 'Setup etcd and add it to the PATH'
```

Comment:
Action says it adds etcd to the PATH, but I don't see it's happening.

Here is how to [add smth to system path](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path)

Reply:
Fixed

Reply:
So you've decided to not setup PATH for etcd. ok then